### PR TITLE
Fix labnotebook description wave not being refetched if changed

### PR DESCRIPTION
**Pull request by t-b**
This was found as part of testing #1380. This fixes the "BUG: ED_FindIndizesAndRedimension" messages. This bug should not influence normal users as you have to acquire data with one MIES version and then acquire data with a newer MIES version again into the same file. Nevertheless worth fixing.

### CODE REVIEW

File: Packages/MIES/MIES_ExperimentDocumentation.ipf [hunk 1]
```diff
@@ -491,7 +491,7 @@ static Function [WAVE colIndizes, variable rowIndex] ED_FindIndizesAndRedimensio
 
 	Make/FREE/D/N=(DimSize(incomingKey, COLS)) indizes = NaN
 
-	WAVE/T/Z desc
+	WAVE/T/ZZ desc
```
Comment (timjarsky) [inline Packages/MIES/MIES_ExperimentDocumentation.ipf:494]: cool!

File: Packages/MIES/MIES_ExperimentDocumentation.ipf [hunk 2]
```diff
@@ -489,7 +489,7 @@ static Function [WAVE colIndizes, variable rowIndex] ED_FindIndizesAndRedimensio
 	numKeyCols = DimSize(key, COLS)
 	lastValidIncomingKeyRow = DimSize(incomingKey, ROWS) - 1
 
-	Make/FREE/U/I/N=(DimSize(incomingKey, COLS)) indizes = NaN
+	Make/FREE/D/N=(DimSize(incomingKey, COLS)) indizes = NaN
```
Comment (timjarsky) [inline Packages/MIES/MIES_ExperimentDocumentation.ipf:492]: @t-b So IP9 converts NaN in unsigned integer wave type, not all wave types, is that right?

**Comment by timjarsky** [comment]
@t-b have you tested this by trying it with different MIES versions or should I do that?

File: Packages/MIES/MIES_ExperimentDocumentation.ipf (hunk 1 above)
Comment (t-b) [reply Packages/MIES/MIES_ExperimentDocumentation.ipf:494]: Jeep, getting into more and more IP9 only features.

File: Packages/MIES/MIES_ExperimentDocumentation.ipf (hunk 2 above)
Comment (t-b) [reply Packages/MIES/MIES_ExperimentDocumentation.ipf:492]: You can not store a NaN in an integer wave (you can only store it in floating point waves), so there has to be some conversion going on. And as the code is as it was difficult to grasp, I've decided to use a double wave instead which can store NaN.

**Comment by t-b** [comment]
> @t-b have you tested this by trying it with different MIES versions or should I do that?

I've done that. Works.

**Comment by timjarsky** [comment]
> > @t-b have you tested this by trying it with different MIES versions or should I do that?
> 
> I've done that. Works.

Thanks!!

**Review by timjarsky — APPROVED** [review]

**Pull request merged**